java-frontend/src/main/java/org/sonar/java/se/symbolicvalues/SymbolicValue.java

```diff
@@ -331,4 +375,8 @@ public String toString() {
   public BinaryRelation binaryRelation() {
     return null;
   }
+
+  protected List<ProgramState> copyAllConstraints(BooleanConstraint booleanConstraint, ProgramState programState) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "booleanConstraint". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1172)